test/components/account/claim.spec.js

```diff
@@ -30,10 +30,9 @@ describe('AccountClaim', () => {
         await component.get('input').setValue('testPassword');
         return component.get('form').trigger('submit');
       })
-      .beforeEachResponse((_, { method, url, headers, data }) => {
+      .beforeEachResponse((_, { method, url, data }) => {
         method.should.equal('POST');
         url.should.equal('/v1/users/reset/verify');
-        headers.Authorization.should.equal('Bearer foo');
```

Comment:
I think we should probably keep this assertion. This isn't a generic assertion about the `Authorization` header and how it's usually set. The `AccountClaim` component does something special and sets the `Authorization` header based on the token in the URL.